bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/JdbcMapper.java

```diff
@@ -131,6 +139,14 @@ public boolean dropItemsTableIfExists(ItemsVO vo) {
         return true;
     }
 
+    public boolean dropTable(String tableName) {
+        logger.debug("JDBC::dropTable");
+        long timerStart = System.currentTimeMillis();
+        conf.getDBDAO().doDropTable(tableName);
+        logTime("doDropTable", timerStart, System.currentTimeMillis());
+        return true;
```

Comment:
Is it necessary to return a boolean rather than void ?

Reply:
Not really. It would be better to throw an exception if it fails, but currently exceptions are swallowed by Yank. This would need some refactoring. Changed to `void`.

Reply:
Addressed in #13719.